packages/mdc-ripple/util.js

```diff
@@ -15,13 +15,21 @@
  */
 
 /** @private {boolean|undefined} */
+let supportsCssVariables_;
+
+/** @private {boolean|undefined} */
 let supportsPassive_;
 
 /**
  * @param {!Window} windowObj
+ * @param {boolean=} forceRefresh
```

Comment:
We have a very similar argument in the util function below this which isn't documented, and I was under the impression it was normal for us to not document these within the jsdoc comments added for Closure?

I did realize that I hadn't added this argument to the README so I added it there.  We don't document it there either though, and I wonder if we should - forceRefresh is only really needed for testing purposes, it shouldn't be needed for normal use.